lib/cocoapods/generator/header.rb

```diff
@@ -82,7 +82,11 @@ def save_as(path)
       # @return [String]
       #
       def generate_platform_import_header
-        "#import #{platform == :ios ? '<UIKit/UIKit.h>' : '<Cocoa/Cocoa.h>'}\n"
+        case platform.name
+        when :ios then "#import <UIKit/UIKit.h>\n"
+        when :osx then "#import <Cocoa/Cocoa.h>\n"
+        else "#import <Foundation/Foundation.h>\n"
```

Comment:
In that case, shouldn't the umbrella header import Foundation and not the pch file?